pkg/controller/storagecluster/reconcile.go

```diff
@@ -931,3 +915,28 @@ func remove(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+func getMonCountOverrideEnvVar() string {
+	return os.Getenv(monOverrideEnvVar)
+}
+
+func getMonCount(nodeCount int) int {
+	count := defaults.MonCount
+
+	// return static value if overriden
+	override := getMonCountOverrideEnvVar()
```

Comment:
I don't think it's worth it having a separate function for this. Just call os.Getenv here.